dot/sync/disjoint_block_set.go

```diff
@@ -4,14 +4,22 @@
 package sync
 
 import (
+	"context"
 	"errors"
 	"math/big"
 	"sync"
+	"time"
 
 	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/lib/common"
 )
 
+const (
+	// ttl is the time that a block can stay in this set before being cleared.
+	ttl                 = time.Minute * 10
```

Comment:
nit call me old school, but shouldn't we put the unit at the end? :smile_cat: 

```suggestion
	ttl                 = 10 * time.Minute
```

Reply:
haha, done